server/commands/list_objects.go

```diff
@@ -0,0 +1,234 @@
+package commands
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"github.com/go-errors/errors"
+	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/pkg/tuple"
+	serverErrors "github.com/openfga/openfga/server/errors"
+	"github.com/openfga/openfga/storage"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/trace"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	MaximumConcurrentChecks = 100 // todo(jon-whit): make this configurable, but for now limit to 100 concurrent checks
+)
+
+type ListObjectsQuery struct {
+	datastore                    storage.OpenFGADatastore
+	logger                       logger.Logger
+	tracer                       trace.Tracer
+	meter                        metric.Meter
+	listObjectsDeadlineInSeconds int
```

Comment:
Duration instead please?

Reply:
The reasoning here was that it wasn't intuitive how to pass a duration as a configuration value

Reply:
It follows the standard Go duration syntax and the `cobra` library standard, which is quite common across the Go community though. If I want a `ms` deadline or a `minute` deadline it's nice to be able to use the standard Go syntax and do `--listObjects-deadline=100ms` or `--listObjects-deadline=2m` instead of calculating the seconds equivalent (which doesn't work for fractions if you want to specify something < 1s).